crates/backend/src/ast.rs

```diff
@@ -36,7 +36,9 @@ pub struct Export {
     pub constructor: Option<String>,
     /// The rust function
     pub function: Function,
+    ///
```

Comment:
We should either not have a doc comment at all, or the comment should have some content.